src/MagnumPlugins/AssimpImporter/AssimpImporter.conf

```diff
@@ -49,6 +49,16 @@ mergeAnimationClips=false
 # key/value pair with the default node transformation
 removeDummyAnimationTracks=true
 
+# Maximum amount of joint weights per vertex to import as vertex attributes.
+# Requires the LimitBoneWeights option to be set as well to take effect.
+# This can be controlled separately for each mesh import.
+maxJointWeights=4
```

Comment:
This only sets `AI_CONFIG_PP_LBW_MAX_WEIGHTS` so I feel like I should list this in the AI_CONFIG_ section and name it appropriately, but `PpLbwMaxWeights` isn't really convincing me :eyes:

Reply:
The comment about setting this per mesh import is also wrong, fixed locally.

Reply:
@mosra Not quite sure how to interpret the thumbs-up 😄 Do I keep it as it?

Reply:
Ah, oh, didn't realize it's about the `LimitBoneWeights` as well. This option alone makes sense to me, wouldn't need a separate `LimitBoneWeights` I think -- instead say that it sets `aiProcess_LimitBoneWeights` as well, and `0` disables any limit.